src/coreclr/jit/llvm.cpp

```diff
@@ -395,6 +393,10 @@ bool Llvm::helperCallHasManagedCallingConvention(CorInfoHelpFunc helperFunc) con
         { FUNC(CORINFO_HELP_CHKCASTCLASS) CORINFO_TYPE_CLASS, { CORINFO_TYPE_PTR, CORINFO_TYPE_CLASS } },
         { FUNC(CORINFO_HELP_CHKCASTANY) CORINFO_TYPE_CLASS, { CORINFO_TYPE_PTR, CORINFO_TYPE_CLASS } },
         { FUNC(CORINFO_HELP_CHKCASTCLASS_SPECIAL) CORINFO_TYPE_CLASS, { CORINFO_TYPE_PTR, CORINFO_TYPE_CLASS } },
+
+        // TODO-LLVM: jithelpers.cpp, this is a bit of a guess...
+        { FUNC(CORINFO_HELP_ISINSTANCEOF_EXCEPTION) CORINFO_TYPE_BOOL, { CORINFO_TYPE_RT_HANDLE, CORINFO_TYPE_CLASS} },
```

Comment:
```suggestion
        { FUNC(CORINFO_HELP_ISINSTANCEOF_EXCEPTION) CORINFO_TYPE_BOOL, { CORINFO_TYPE_PTR, CORINFO_TYPE_CLASS } },
```
Implementation: https://github.com/dotnet/runtime/blob/2ee941b5e33b68fc39d2c6c2a26f025e0d2efee6/src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/TypeCast.cs#L932.